activation/activation.go

```diff
@@ -485,12 +491,17 @@ func (b *Builder) PublishActivationTx(ctx context.Context) error {
 	}
 
 	atx := b.pendingATX
+	if err := b.SignAtx(atx); err != nil {
+		b.log.Error(err.Error())
```

Comment:
i think caller of the PublishActivationTx will log the error, no need to log it here, especially without any context

Reply:
This was added to be in line with the previous code. If broadcast fails a few lines further down it also logs the error. Shall I remove both logging calls and just return the error, or change it only here?

Reply:
this is completely up to you, i just commented on it because this code was changed in this pr